### PR TITLE
Fix separated shipping address

### PR DESCRIPTION
**Pull request by artursmet**
Fixed separated shipping address form

### CODE REVIEW

File: saleor/checkout/steps.py [hunk 1]
```diff
@@ -157,13 +158,19 @@ def __init__(self, request, storage, cart,
         self.forms['delivery'] = DeliveryForm(
             delivery_choices, request.POST or None,
             initial={'method': selected_method_name})
+        self.shipping_same_as_billing = request.POST.get(
```
Comment (patrys) [inline saleor/checkout/steps.py]: Don't access `request.POST` directly.

File: saleor/checkout/steps.py [hunk 2]
```diff
@@ -136,6 +136,7 @@ def __init__(self, request, storage, cart,
                  default_address=None):
         self.cart = cart
         address_data = storage.get('address', {})
+        self.billing_address = default_address
```
Comment (patrys) [inline saleor/checkout/steps.py]: Rename `default_address` to `shipping_address` as it's no longer a mere default.

**Comment by artursmet** [comment]
@patrys Fixed coverage issues and ready for review.

File: saleor/static/js_src/storefront.jsx [hunk 3]
```diff
@@ -227,7 +227,7 @@ class FormShippingToggler extends React.Component {
     render() {
         return <div className="checkbox">
             <label>
-                <input checked={this.state.value} type="checkbox" onChange={this.formFullToggle} name="shipping_same_as_billing" />
+                <input checked={this.state.value} type="checkbox" onChange={this.formFullToggle.bind(this)} name="shipping_same_as_billing" />
```
Comment (patrys) [inline saleor/static/js_src/storefront.jsx]: `{::this.formFullToggle}`?

**Pull request merged**